app/frontend/src/components/commons/Modal/InputModal.tsx

```diff
@@ -0,0 +1,37 @@
+import { Input } from '@/components/commons/Input';
+import { sansRegular16 } from '@/styles/font.css';
+
+import { Footer } from './Footer';
+import * as styles from './index.css';
+
+type InputModalProps = {
+  description: string;
+  confirmButtonText?: string;
+  cancelButtonText?: string;
+  onClickConfirm: () => void;
+};
+
+export function InputModal({
+  description,
+  confirmButtonText = '확인',
+  cancelButtonText = '취소',
+  onClickConfirm,
+}: InputModalProps) {
+  return (
+    <>
+      <div className={styles.background} />
+      <dialog className={styles.container}>
```

Comment:
```suggestion
        <dialog className={styles.container} open ref={dialogRef}>
```

Open 속성과 ref를 활용해 모달을 닫는 메소드를 작성하지 않고 모달을 조작할 수 있을 것 같아요.